truffle/test/04_workload.js

```diff
@@ -43,31 +41,31 @@ describe('Workload', async () => {
       });
       sentFundTxReceiptPromises.push(txReceiptPromise);
     }
-    
+
     while(sentFundTxReceiptPromises.length) {
       const txReceiptId = await sentFundTxReceiptPromises.pop();
```

Comment:
These are NOT txreceiptIds anymore, they are actual Receipts. imo you should have merge both `while` into a single one as the second one do not improve the performance of the execution and it bring a unnecesary overhead

Reply:
yea i know, was late friday hour and it was working so I was lazy. Merged.